table/server/underdb/hive/src/main/java/alluxio/table/under/hive/util/HiveClientPool.java

```diff
@@ -79,20 +73,7 @@ protected IMetaStoreClient createNewResource() throws IOException {
 
       IMetaStoreClient client = HMSClientFactory.newInstance(
           RetryingMetaStoreClient.getProxy(conf, NOOP_HOOK, HiveMetaStoreClient.class.getName()));
```

Comment:
I think it works just as well to let other exception handling handle db not exist errors than explicitly checking for it. 
@gpang what do you think?

Reply:
I thought maybe there was a reasons to explicitly check for the db, but I forgot why...